server/test/unit/server/db/model/__init__.py

```diff
@@ -0,0 +1,28 @@
+"""
+Tests for the pulp.server.db.model module.
+"""
+from datetime import datetime
+import unittest
+
+import mock
+
+
+class TestCeleryBeatLock(unittest.TestCase):
+    """
+    Test the CeleryBeatLock class.
+    """
+    @mock.patch('pulp.server.db.model.__init__', new=mock.Mock())
+    @mock.patch('pulp.server.db.model.CeleryBeatLock')
```

Comment:
Generally not a good idea to mock the class you are testing.

The long term plan for the files in `pulp/srver/db/model/*` was that they will all be converted to mongoengine and removed. We are changing that plan slightly, but it doesn't really matter. The point is that the `model/__init__.py` tests are in `pulp/server/db/test_model.py`. There is not much to test with these since we are letting Mongoengine do the work, but take a look at the other model tests to see what is asserted.